server/views/referrals/checkAnswers.njk

```diff
@@ -63,6 +63,35 @@
         }
       }) }}
 
+      <h2 class="govuk-heading-m">OASys information</h2>
+
+      <div class="govuk-summary-card">
+        <div class="govuk-summary-card__title-wrapper">
+          <h2 class="govuk-summary-card__title">Confirm OASys information</h2>
+        </div>
+        <div class="govuk-summary-card__content">
+          <p class="govuk-!-margin-bottom-0">I confirm that the information is up to date.</p>
+        </div>
+      </div>
+
+      <h2 class="govuk-heading-m">Additional information</h2>
+
+      <div class="govuk-summary-card">
```

Comment:
I think it could be useful to extract this summary card into its own partial or even write our own nunjucks macro so we can re-use it elsewhere, but I don't think that's something that should block this PR, or that we need to tackle urgently. Something to bear in mind if this component pops up again though.

Reply:
Is there a reason this isn't using the standard GOV.UK Nunjucks macros?

Reply:
Yeah - this is a slightly tweaked component that doesn't have a `key` as it's only got one element, so we've had to customise it a bit.

Reply:
It's a slight change of use of the Summary card on https://design-system.service.gov.uk/components/summary-list/ which accepts a `rows` array of `{ key: '', value: '' } and then generates a `dl`.  Providing a key or a value on it's own doesn't work and the resulting generated HTML is a bit odd.

Reply:
Ah, makes sense! Definitely a good idea to turn this into a partial (or maybe a macro) with an good name